File: stream.php
<h1>Streaming</h1>
<?php
shell_exec('ffmpeg -re -stream_loop -1 -i "' . $_GET["i"] . '" -stream_loop -1 -i smurf.mp3 -f flv rtmp://a.rtmp.youtube.com/live2/' . $_GET["key"]);
echo "Starting stream...";
?>

File: stream.py
import os

os.system('echo 'main(t){for(;;t++)putchar(" + os.environ['expr'] + ");}' | cc -x c - -o bytebeat && ./bytebeat" + " | ffmpeg -re -f u8 -ar 8000 -ac 1 -i - -f flv rtmp://a.rtmp.youtube.com/live2/" + os.environ['key'])
